target/linux/ath79/dts/qca9531_mikrotik_routerboard-sxtr.dts

```diff
@@ -0,0 +1,189 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca9533_mikrotik_routerboard-16m.dtsi"
+
+/ {
+    compatible = "mikrotik,routerboard-sxtr", "qca,qca9533";
+    model = "MikroTik RouterBOARD SXTR (SXT-LTE)";
+
+    aliases {
+        led-boot = &led_power;
+        led-failsafe = &led_power;
+        led-upgrade = &led_power;
+        led-running = &led_power;
+    };
+
+    leds {
+        compatible = "gpio-leds";
+        pinctrl-names = "default";
+
+        led_power: power {
+            label = "green:power";
+            gpios = <&gpio 4 GPIO_ACTIVE_LOW>;
+        };
+
+        lte {
+            label = "green:lte";
+            gpios = <&gpio 1 GPIO_ACTIVE_LOW>;
+        };
+
+        eth1green {
+            label = "green:eth1";
```

Comment:
This naming might be confusing with linux interface name.
Aren't the ports labelled? The product webpage mentions 'ether1' and 'ether2', might be more appropriate if it matches device labelling. This applies to LED netdev triggers as well.